src/System.ComponentModel.Annotations/src/System/ComponentModel/DataAnnotations/EmailAddressAttribute.cs

```diff
@@ -32,7 +25,10 @@ public override bool IsValid(object value)
             }
 
             var valueAsString = value as string;
-            return valueAsString != null && _regex.Match(valueAsString).Length > 0;
+            return (valueAsString != null
+                && valueAsString.Count(c => c == '@') == 1
+                && valueAsString[0] != '@'
+                && valueAsString[valueAsString.Length-1] != '@');
```

Comment:
My guess is it would not be an issue but maybe there's a user out there who is doing what you suggest. I'll change it anyway just to be safe.